db/schema.rb

```diff
@@ -123,8 +123,8 @@
   create_table "distributors_shipping_methods", :force => true do |t|
     t.integer  "distributor_id"
     t.integer  "shipping_method_id"
-    t.datetime "created_at",         :null => false
-    t.datetime "updated_at",         :null => false
+    t.datetime "created_at"
+    t.datetime "updated_at"
```

Comment:
Had manually to unstage those changes. As `rake db:schema:load` throws PostgreSQL error. and `rake db:migrate:up VERSION={MIGRATION_ID}` still removes those `:null => false`. So those suggestions do not work.